package.json

```diff
@@ -128,7 +128,7 @@
     "check:atlas": "mocha --config test/manual/mocharc.json test/manual/atlas_connectivity.test.js",
     "check:adl": "mocha --config test/mocha_mongodb.json test/manual/atlas-data-lake-testing",
     "check:aws": "mocha --config test/mocha_mongodb.json test/integration/auth/mongodb_aws.test.ts",
-    "check:oidc": "mocha --config test/manual/mocharc.json test/manual/mongodb_oidc.prose.test.ts",
+    "check:oidc": "TRACE_SOCKETS=true mocha --config test/manual/mocharc.json test/manual/mongodb_oidc.prose.test.ts",
```

Comment:
Unfortunately the manual test config doesn't bring in the leak checker

you can add 
```json
"require": [
    "test/tools/runner/hooks/leak_checker.ts"
]
```
to `test/manual/mocharc.json`

Reply:
Going to remove that as I found the leaked socket.

Reply:
Done